src/crypto/BUILD.gn

```diff
@@ -51,6 +51,8 @@ static_library("crypto") {
     "CHIPCryptoPAL.h",
   ]
 
+  cflags = [ "-Wconversion" ]
```

Comment:
once everything is `-Wconversion`-kosher, this gets moved up?

Reply:
Yes, the plan is to make it treewide once the whole tree is clean, but doing it all in one PR was not really going to be workable.